<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TEM Autosize Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QGroupBox" name="groupInput">
          <property name="title">
           <string>Input</string>
          </property>
          <layout class="QVBoxLayout" name="inputLayout">
           <item>
            <layout class="QHBoxLayout" name="inputButtonsLayout">
             <item>
              <widget class="QPushButton" name="btnAddImages">
               <property name="text">
                <string>Add Images</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnClearList">
               <property name="text">
                <string>Clear List</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="listImages"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupScaleBar">
          <property name="title">
           <string>Scale Bar</string>
          </property>
          <layout class="QVBoxLayout" name="scaleLayout">
           <item>
            <layout class="QHBoxLayout" name="sbColorLayout">
             <item>
              <widget class="QRadioButton" name="radioWhite">
               <property name="text">
                <string>White</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioBlack">
               <property name="text">
                <string>Black</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sbLengthLayout">
             <item>
              <widget class="QLabel" name="labelSbDefault">
               <property name="text">
                <string>Default length (nm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinSbDefault">
               <property name="minimum">
                <double>0.1</double>
               </property>
               <property name="maximum">
                <double>10000.0</double>
               </property>
               <property name="value">
                <double>20.0</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupThresholds">
          <property name="title">
           <string>Thresholds</string>
          </property>
          <layout class="QGridLayout" name="thresholdsLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelMinNm">
             <property name="text">
              <string>Min size (nm)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="spinMinNm">
             <property name="minimum">
              <double>0.1</double>
             </property>
             <property name="maximum">
              <double>1000.0</double>
             </property>
             <property name="value">
              <double>3.5</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelMaxNm">
             <property name="text">
              <string>Max size (nm)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="spinMaxNm">
             <property name="minimum">
              <double>0.1</double>
             </property>
             <property name="maximum">
              <double>1000.0</double>
             </property>
             <property name="value">
              <double>9.0</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelEcc">
             <property name="text">
              <string>Eccentricity max (Lower values select more circular particles)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="spinEcc">
             <property name="minimum">
              <double>0.0</double>
             </property>
             <property name="maximum">
              <double>1.0</double>
             </property>
             <property name="singleStep">
              <double>0.05</double>
             </property>
             <property name="value">
              <double>0.7</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelSol">
             <property name="text">
              <string>Solidity min (Higher values select more solid, fully filled particles)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="spinSol">
             <property name="minimum">
              <double>0.0</double>
             </property>
             <property name="maximum">
              <double>1.0</double>
             </property>
             <property name="singleStep">
              <double>0.05</double>
             </property>
             <property name="value">
              <double>0.5</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupActions">
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QVBoxLayout" name="actionsLayout">
           <item>
            <widget class="QPushButton" name="btnAnalyze">
             <property name="text">
              <string>Analyze Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAccept">
             <property name="text">
              <string>Accept and Accumulate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSkip">
             <property name="text">
              <string>Skip Image</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupSession">
          <property name="title">
           <string>Session</string>
          </property>
          <layout class="QVBoxLayout" name="sessionLayout">
           <item>
            <layout class="QHBoxLayout" name="sessionTopLayout">
             <item>
              <widget class="QLabel" name="labelTotal">
               <property name="text">
                <string>Total: 0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnExport">
               <property name="text">
                <string>Export CSV</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sessionStatsLayout">
             <item>
              <widget class="QLabel" name="labelMean">
               <property name="text">
                <string>Mean: -</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelStd">
               <property name="text">
                <string>Std: -</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sessionImageStatsLayout">
             <item>
              <widget class="QLabel" name="labelImgCount">
               <property name="text">
                <string>Image count: -</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelImgMean">
               <property name="text">
                <string>Image mean: -</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelImgStd">
               <property name="text">
                <string>Image std: -</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightPanel">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QGroupBox" name="groupPreview">
          <property name="title">
           <string>Preview</string>
          </property>
          <layout class="QVBoxLayout" name="previewLayout">
           <item>
            <widget class="QLabel" name="labelPreview">
             <property name="text">
              <string>Image preview will appear here.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>380</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupHistogram">
          <property name="title">
           <string>Histogram</string>
          </property>
          <layout class="QVBoxLayout" name="histLayout">
           <item>
            <layout class="QHBoxLayout" name="histControlsLayout">
             <item>
              <widget class="QLabel" name="labelHistBins">
               <property name="text">
                <string>Bins</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinHistBins">
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="value">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkHistRange">
               <property name="text">
                <string>Use range</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelHistMin">
               <property name="text">
                <string>Min</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinHistMin">
               <property name="minimum">
                <double>0.0</double>
               </property>
               <property name="maximum">
                <double>10000.0</double>
               </property>
               <property name="value">
                <double>0.0</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelHistMax">
               <property name="text">
                <string>Max</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="spinHistMax">
               <property name="minimum">
                <double>0.0</double>
               </property>
               <property name="maximum">
                <double>10000.0</double>
               </property>
               <property name="value">
                <double>10.0</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="histPlaceholder"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupLog">
         <property name="title">
          <string>Log</string>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>180</height>
          </size>
         </property>
          <layout class="QVBoxLayout" name="logLayout">
           <item>
            <widget class="QPlainTextEdit" name="textLog">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
